Source/WarpX.cpp

```diff
@@ -531,6 +532,10 @@ WarpX::ReadParameters ()
         pp.query("pml_has_particles", pml_has_particles);
         pp.query("do_pml_j_damping", do_pml_j_damping);
         pp.query("do_pml_in_domain", do_pml_in_domain);
+
+    // Read the value of the time advancement scheme of M field
+        pp.query("time_scheme_order", time_scheme_order);
```

Comment:
we should renname time_scheme_order to something more illustrative of the fact it's for the M field
LLG_time_scheme_order or mag_time_scheme_order.

Also this should be wrapped in an #ifdef WARPX_MAG_LLG, as well as all usages of this.

Reply:
I prefer mag_time_scheme_order.  @Reva and @jackie, what do you think?

Reply:
either name works well for me :-)